content/docs/faq-state.md

```diff
@@ -18,6 +18,75 @@ Here are some good resources for further reading on when to use `props` vs `stat
 * [Props vs State](https://github.com/uberVU/react-guide/blob/master/props-vs-state.md)
 * [ReactJS: Props vs. State](http://lucybain.com/blog/2016/react-state-vs-pros/)
 
+### Is synching state and props a bad idea?
+
+In rare cases, it's okay to initialize state based on props. One valid use to store props in state is to be able to know its previous values, because props can change over time. This effectively "forks" the props and sets the state with the initial props.
+
+Here's an example of what not to do:
```

Comment:
It's a bit confusing we start with "in rare cases it's okay" but then the example shows when it's not okay. Let's change to start with the "bad" section and only mention the good use case at the end.